src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/TravellerSelection.tsx

```diff
@@ -182,60 +193,85 @@ export function TravellerSelection({
             : t(PurchaseOverviewTexts.travellerSelection.titleNotSelectable)
         }
       />
-      <Section {...accessibility}>
```

Comment:
The naming doesn't really make sense since we now have a `TravellerSeletionWrapper` inside the `TravellerSelection` component. So should find a better name for it.

How do you think it looks the other way around, without introducing a new component? Like this ish? (pseudocode)

```tsx
const content = (
   <View style={styles.sectionContentContainer}>
          <View style={{flex: 1}}>
            <ThemeText type="body__primary--bold">
              {multipleTravellerCategoriesSelectedFrom
  ...
);
```

and then:
```tsx
<Section {...props}>
  {canSelectUserProfile
    ? <GenericClickableSectionItem
        onPress={travellerSelectionOnPress}
        ref={ref}
    >
      {content}
    </GenericClickableSectionItem>)
    : <GenericSectionItem>{content}</GenericSectionItem>}
</Section>
```

Reply:
I think that should also work, I'll try 👍

Reply:
Made the change, looks better now I think, thank you!